sr_point_cloud/test/cloud_cropping.test

```diff
@@ -0,0 +1,11 @@
+<launch>
+  <include file="$(find freenect_launch)/launch/freenect.launch"/>
```

Comment:
this is hopefully not needed since you're using a bag file (otherwise the automatic tests needs an actual kinect or equivalent to run?